src/components/cesium_map/config_cesium.js

```diff
@@ -0,0 +1,8 @@
+// color bind schema Wong
+const colorbind = ["#D55E00", "#E69F00", "#009E73", "#56B4E9", "#CC79A7", "#F0E442"];
+const source = ["GEOME", "OPENCONTEXT", "SESAR", "SMITHSONIAN"];
```

Comment:
@dannymandel Do you think we should put the global constant parameters to the root `config.js` or just put them in the Cesium folder?

Reply:
Ideally the values of the sources should be retrieved from the search index